### PR TITLE
MyCart Page

### PR DESCRIPTION
**Pull request by isaacruz2308**
Just created the MyCart Page and changed the path of the nav bar button so it takes users to the page

### CODE REVIEW

File: frontend/src/pages/customers/cart/index.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            ${(item.price * item.quantity).toFixed(2)}
+          </Typography>
+        </Stack>
+      </CardContent>
+    </Card>
+  );
+};
+
+export default function MyCartPage() {
+  const [items, setItems] = useState<CartItemArray>(cartItems);
```
Comment (Kenneth-Aponte) [inline frontend/src/pages/customers/cart/index.tsx]: lets try and use a callback function, like on like 32

**Review by Kenneth-Aponte — COMMENTED** [review]
Code looks good, I know it will be a big change but, i think we could use the [mui table component](https://mui.com/material-ui/react-table/) instead so that everything is aligned and it will be easier to handle when we connect this to the backend. 

Using the basic table, something like this:
Screenshot 2024-03-16 at 11 37 14 AM

let me know what you think :)

File: frontend/src/pages/customers/cart/index.tsx (hunk 1 above)
Comment (isaacruz2308) [reply frontend/src/pages/customers/cart/index.tsx]: Will follow

Comment (isaacruz2308) [reply frontend/src/pages/customers/cart/index.tsx]: Are we sure we want to use a callback function for this page component? Veo que usan callback en el Navbar component, pero en las paginas de Menu y Profile lo estan haciendo con el mismo metodo que use. I was just trying to keep the code consistent. Let me know what you think ^^

Comment (Kenneth-Aponte) [reply frontend/src/pages/customers/cart/index.tsx]: I see, you're right, I thought we were using callback on all of the components. You can leave it as is or with a callback :)

**Review by Kenneth-Aponte — APPROVED** [review]

**Comment by Kenneth-Aponte** [comment]
@isaacruz2308 looked at the pr again and its ready, you can merge cuando quieras :)

**Pull request merged**